doc/CMakeLists.txt

```diff
@@ -488,3 +488,63 @@ add_custom_target(
   COMMAND ${CMAKE_MAKE_PROGRAM} nrf
   COMMAND ${CMAKE_MAKE_PROGRAM} nrfxlib
 )
+
+
+#
+# Add targets for checking links
+#
+
+add_custom_target(
+  nrf-linkcheck
```

Comment:
So does this mean that if any of the Links stop working then the CI build will have errors, and CI will be red?

Reply:
No. This target isn't used in the CI (at least not for now). 
The plan is to run it locally before a release to confirm that the links work - up to now, we did that manually by clicking all links. ;)